packer/beta-vars.json

```diff
@@ -0,0 +1,10 @@
+{
+  "semester": "Sp20",
+  "mint_version": "19.3",
+
+  "git_repo": "https://github.com/jmunixusers/cs-vm-build",
```

Comment:
Since these are the defaults, do they need to be overriden in this file?

Reply:
I fiddle with the repo and branch enough during beta testing that it would be helpful to have it present.